app/src/main/java/com/feedhenry/securenativeandroidtemplate/domain/models/Note.java

```diff
@@ -0,0 +1,40 @@
+package com.feedhenry.securenativeandroidtemplate.domain.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by weili on 08/09/2017.
+ */
+
+public class Note {
+
+    private String id;
+    private String title;
+    private String content;
+    protected Date createdAt;
+
+    public Note(String title, String content) {
+        this.createdAt = new Date();
+        this.id = UUID.randomUUID().toString();
```

Comment:
Might be a good opportunity to use `java.security.SecureRandom` to lower the threat score.

Reply:
I think it's a bit overkill, given that the id is not really security related? It's just an identify field

Reply:
True, if we are going to use this secure random function somewhere else in the app then we can leave it out here. Just trying to think of a place to actually use it at least once or else it will get flagged in the report. Maybe we could just have a simple button that shows a random value on screen somewhere else in the app instead later on.